llvm/lib/Support/CommandLine.cpp

```diff
@@ -404,6 +404,22 @@ class CommandLineParser {
 
 static ManagedStatic<CommandLineParser> GlobalParser;
 
+template <typename T, T TrueVal, T FalseVal>
+static bool parseBool(Option &O, StringRef ArgName, StringRef Arg, T &Value) {
+  if (Arg == "" || Arg == "true" || Arg == "TRUE" || Arg == "True" ||
```

Comment:
We could use `Arg.equals_insensitive("true") here. Same below for `false`

Reply:
But that would match more than what the code current matches. For example, tRue or TrUE.